controller_manager:
  ros__parameters:
    update_rate: 500  # Hz

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

joint_trajectory_controller:
  ros__parameters:
    position_proportional_gain: 2.0
    joints:
    - j_1_1
    - j_1_2
    - j_1_3
    - j_1_4
    - j_2_1
    - j_2_2
    - j_2_3
    - j_2_4
    - j_3_1
    - j_3_2
    - j_3_3
    - j_3_4
    - j_4_1
    - j_4_2
    - j_4_3
    - j_4_4
    - j_5_1
    - j_5_2
    - j_5_3
    - j_5_4
    
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity